static/main.css

```diff
@@ -0,0 +1,43 @@
+#header{
+    border-style: solid ;
+    border-color: black;
+    background-color:lightblue;
+    width: auto;
+    margin-bottom: 3%;
+    height: 50px;
```

Comment:
We should change height to some % instead of px since we have used % throughout the file

Reply:
Got it